javagen/src/main/java/com/azure/autorest/util/TemplateUtil.java

```diff
@@ -59,6 +59,14 @@ public class TemplateUtil {
 
     public static final String DATE_UTC = "date-utc";
 
+    public static final String GENERATED_JAVADOC_DESC_START_MARKER = "<!-- start generated doc description -->";
+
+    public static final String GENERATED_JAVADOC_DESC_END_MARKER = "<!-- end generated doc description-->";
+
+    public static final String GENERATED_JAVADOC_TAG_START_MARKER = "<!-- start generated doc tags -->";
+
+    public static final String GENERATED_JAVADOC_TAG_END_MARKER = "<!-- end generated doc tags-->";
```

Comment:
TemplateUtil is being used for all utilities and will make it hard to find things. So, I suggest we have a JavadocUtil or something similar to group utilities that are more specific.